src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -752,7 +752,13 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
         ) as DefaultVideoTile;
         const paused = videoTile?.state().paused || false;
         if (!chosenStreams.some(stream => stream.attendeeId === preference.attendeeId)) {
-          if (videoTile) {
+          // We cannot rely on the existance of video tile to indicate that the source exists in the call
+          // because tiles will not be added or removed until after a full renegotiation (i.e. it will
+          // be behind the state provided by the index)
+          const streamExists = remoteInfos.some(
+            stream => stream.attendeeId === preference.attendeeId
+          );
+          if (videoTile && streamExists) {
```

Comment:
Should we log an info log here that there is no stream exists.

Reply:
This is technically normal behavior and is hit everytime a video is removed.  Think a log may be overkill since we already log the full index.